include/yaramod/parser/parser_driver.h

```diff
@@ -253,9 +253,9 @@ class ParserDriver
 	/// @name Constructors
 	/// @{
 	ParserDriver() = delete;
-  	ParserDriver(ParserMode parserMode);
-	explicit ParserDriver(const std::string& filePath, ParserMode parserMode = ParserMode::Regular);
-	explicit ParserDriver(std::istream& input, ParserMode parserMode = ParserMode::Regular);
+  	ParserDriver(ParserMode parserMode, bool avastSpecific = true, bool vtSpecific = true);
```

Comment:
It would be best to avoid just boolean parameters if it's unclear from the context of the function name of what they suppose to mean. Calls to such functions/methods then look like this `ParserDriver(..., true, false)`. What exactly those `true` and `false` mean? The best way to do it would be through bitmasks because then you can do something like `ParserDriver(..., Features::VirusTotal | Features::Avast)`.

Reply:
OK, I implemented this in the last commit.

Reply:
OK, I implemented this in the last [commit](https://github.com/avast/yaramod/pull/51/commits/591e8499aeced53445819fa2b3602219cb4645a5).